### ISSUE TITLE
Export empty groups (transforms) or a joint that is not used for deformation

### ISSUE THREAD

**Issue by BigRoy** [iimachines/Maya2glTF#163]
### Issue

I seem to be unable to export a null object, like a group/locator/joint - I'd like to export an additional animated transform in the hierarchy.

Is there a way to do this?

**Comment by Ziriax** [comment]
You can try to explicitly select these objects. Then they should be included in the export. If not, this is a bug I guess.

**Comment by BigRoy** [comment]
I can't seem to get them to be exported, they also don't show in the `dumpMaya` output at all.

It might be related to [this code only processing Meshes and Cameras](https://github.com/iimachines/Maya2glTF/blob/3d7ba8eaf03ea15afe0720129db88deba7f8749a/src/ExportableAsset.cpp#L55-L65) nor does it get processed as ExportableNode since it doesn't trigger [this mixed pivot point warning](https://github.com/iimachines/Maya2glTF/blob/3d7ba8eaf03ea15afe0720129db88deba7f8749a/src/ExportableNode.cpp#L51-L56) on a Transform that I explicitly gave two different pivots. That warning does show as soon as I parent a mesh underneath and export the mesh instead.

Where should I be looking in the code to enable exporting selected nodes explicitly?

**Comment by Ziriax** [comment]
Indeed, it seems to be missing code to also export separate nodes that are not part of skeleton...

It should be easy to add these I guess, just an extra loop over locator nodes.

**Comment by BigRoy** [comment]
> It should be easy to add these I guess, just an extra loop over locator nodes.

That's what I was hoping. :) It's hard to pinpoint where it's logical the relevant code gets added. [Since even already in this Arguments class it's filtered to shapes or cameras](https://github.com/iimachines/Maya2glTF/blob/3d7ba8eaf03ea15afe0720129db88deba7f8749a/src/Arguments.cpp#L704-L773) for which a locator or empty null group is neither.

Should almost have a `args.otherDagNodes` or alike. You might be more familiar with the code base, what seems like the most logical approach?

**Comment by Ziriax** [comment]
I think in that code it is just missing an extra if block that checks iMFn::kLocator...

It has been a long time since I worked on the plugin ;-)

It does make sense to add a flag to remain backwards compatible. 

Do want to give it a shot yourself?

I am about to leave on vacation for 3 weeks unfortunately...
